### ISSUE TITLE
Type in verse markers while typing

### ISSUE THREAD

**Issue by RRohrssen** [unfoldingWord-dev/ts-desktop#576]
Tom has a team of typists. The typists take the handwritten work that was completed during a MAST event and type them into translationStudio. Their workflow seems to be interrupted and the team frustrated by having the verse markers reset and manually placing them in again and again after every edit. They would like to be able to just type the numbers in while typing the verse in and not having them reset later.

**Comment by cdwhitfield73** [comment]
why did this fail? @cckozie

**Comment by cckozie** [comment]
Sorry @cdwhitfield73: I wrote the comment but apparently failed to click the "Comment" button.
Prior to 8.0:
If I edit a chunk that has 2 verses and I set the second verse marker to its intended place, and then mark the chunk as done, the first verse marker appears at the beginning of the text and the other one as expected.
But in 8.0:
If I edit the chunk and key in the marker for the second one but not the first, when I exit edit mode I see the verse marker bubbles where I would expect to see them, but after marking the chunk as done I do not see a verse marker for the first one at the beginning of the text as before. To fix it now, before marking the chunk as done I need to either key in the first verse marker or "move" it to where it already is.
